dawn/renderer/components/Editor.js

```diff
@@ -439,6 +439,12 @@ class Editor extends React.Component {
               onClick={this.upload}
               glyph="upload"
             />
+            <TooltipButton
```

Comment:
I think maybe add button disabled case, because the generator that handles `onDownloadCode` doesn't "run" unless
`stateSlice.runtimeStatus && stateSlice.ipAddress !== defaults.IPADDRESS`.
This might be somewhat helpful for the student UI-wise.